Mods/interface/PuppiMod.h

```diff
@@ -101,6 +106,8 @@ namespace mithep
       std::vector<Double_t> fMedEtaSFs;              // Scale factor for median as a function of Eta
       std::vector<Double_t> fEtaMaxExtraps;          // I think this is the maximum eta to calculate median alphas?
 
+      ParticleMapper* fMapper;                        // For efficient determination of particle proximity
```

Comment:
See the discussion on the ParticleMapper later, but basically with a few modifications, we could reuse a single mapper for all events, and that is more CPU-efficient (because each time you initialize the mapper you need to allocate memory for all the eta-phi bins).